# 以相機座標系統到無人機座標系統的轉換，單位是公尺及度
# 以無人機為原點
camera_x: 0.0
camera_y: -0.125
camera_z: 0.0
camera_roll: 0.0
camera_pitch: 0.0
camera_yaw: 0.0